Avoid having to pull in dev-python/coverage test dep since we don't care about
coverage.

--- binwalk-2.2.0/setup.py
+++ binwalk-2.2.0/setup.py
@@ -303,7 +303,7 @@
         os.chdir(testing_directory)
 
         # Run the tests
-        retval = nose.core.run(argv=['--exe','--with-coverage'])
+        retval = nose.core.run(argv=['--exe'])
 
         sys.stdout.write("\n")
 
